[2023-12-11 18:38:23,401] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:23,404] INFO _log() L96    * Restarting with stat - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:40,172] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:40] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:40,272] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:40] "GET /openapi/static/css/swagger-ui.css HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:40,293] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:40] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:40,297] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:40] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:40,432] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:40] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:38:46,376] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:38:46] "GET /vinhos HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:39:10,875] INFO _log() L96   192.168.68.105 - - [11/Dec/2023 18:39:10] "DELETE /vinho?name=Amostra_1 HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:43:53,365] WARNING get_vinhos() L51   Não há vinhos cadastrados na base :/ - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\app.py L51  
[2023-12-11 18:43:53,366] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:43:53] "[33mGET /vinhos?name=Amostra_1 HTTP/1.1[0m" 404 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:43:53,421] ERROR log_exception() L825  Exception on /vinho [POST] - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask\app.py L825 
Traceback (most recent call last):
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask\app.py", line 1455, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask\app.py", line 869, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask\app.py", line 867, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask\app.py", line 852, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\app.py", line 68, in predict
    modelo = Model.carrega_modelo(ml_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Model.carrega_modelo() missing 1 required positional argument: 'path'
[2023-12-11 18:43:53,428] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:43:53] "[35m[1mPOST /vinho HTTP/1.1[0m" 500 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:45:06,491] INFO _log() L96    * Detected change in 'C:\\Users\\malve\\source\\repos\\mvp-sprint-3-dataset\\wine_classification_api\\model\\modelo.py', reloading - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:45:06,688] INFO _log() L96    * Restarting with stat - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:45:12,918] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:45:12] "OPTIONS /vinho?name=Amostra_1 HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:45:12,928] WARNING delete_vinho() L167  Erro ao deletar vinho 'Amostra_1', Vinho não encontrado na base :/ - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\app.py L167 
[2023-12-11 18:45:12,928] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:45:12] "[33mDELETE /vinho?name=Amostra_1 HTTP/1.1[0m" 404 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:46:56,434] WARNING get_vinhos() L51   Não há vinhos cadastrados na base :/ - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\app.py L51  
[2023-12-11 18:46:56,434] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:46:56] "[33mGET /vinhos?name=Amostra_1 HTTP/1.1[0m" 404 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:46:56,478] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:46:56] "POST /vinho HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:47:50,272] INFO _log() L96    * Detected change in 'C:\\Users\\malve\\source\\repos\\mvp-sprint-3-dataset\\wine_classification_api\\model\\modelo.py', reloading - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:47:50,438] INFO _log() L96    * Restarting with stat - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:49:03,387] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:49:03] "OPTIONS /vinho?name=Amostra_1 HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:49:03,407] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:49:03] "DELETE /vinho?name=Amostra_1 HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:49:06,546] WARNING get_vinhos() L51   Não há vinhos cadastrados na base :/ - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\app.py L51  
[2023-12-11 18:49:06,547] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:49:06] "[33mGET /vinhos?name=Amostra_1 HTTP/1.1[0m" 404 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:49:06,690] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:49:06] "[35m[1mPOST /vinho HTTP/1.1[0m" 500 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:50:45,297] INFO _log() L96   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.68.105:5000 - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:50:45,298] INFO _log() L96   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:50:45,299] INFO _log() L96    * Restarting with stat - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
[2023-12-11 18:51:05,891] INFO _log() L96   127.0.0.1 - - [11/Dec/2023 18:51:05] "OPTIONS /vinho?name=Amostra_1 HTTP/1.1" 200 - - call_trace=C:\Users\malve\source\repos\mvp-sprint-3-dataset\wine_classification_api\venv\Lib\site-packages\werkzeug\_internal.py L96  
